### DISCUSSION TITLE
Dark mode addition

### DISCUSSION THREAD

**Discussion by singodiyashubham87** [karmacircleCommunity/KarmaCircle#1022] [Ideas]
I guess we should add an option for dark mode to the website.
Most websites nowadays already give this feature and most of the users prefer dark mode rather than light mode, So it will be better for the UX if we introduce dark mode.

@tamalCodes   @karthiknadar1204 @Nishitbaria  What are your thoughts?

**Comment by karthiknadar1204** [comment]
But considering the niche of the site,dark mode just doesn't add up is what I think.

**Reply by Nishitbaria** [reply]
I believe the same applies when we have documentation and users are reading for extended periods in such cases, implementing a dark mode might be beneficial. 👀

**Comment by singodiyashubham87** [comment]
> But considering the niche of the site,dark mode just doesn't add up is what I think.

@karthiknadar1204 That's a valid point by the way.
@tamalCodes What you say?

**Reply by tamalCodes** [reply]
True, i think our site which has a theme of orange and black is not gonna look proper in black, however we need to brain strorm and make something for the dark mode too, as it is TBH very painful to eyes to stare at white screens.
